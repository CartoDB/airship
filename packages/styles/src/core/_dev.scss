@import 'tools/_mixins';
@import 'variables/_variables';
