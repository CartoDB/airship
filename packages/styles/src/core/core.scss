@import 'variables/_variables';

* {
  box-sizing: border-box;
  font-family: sans-serif;
}

:root {
  --primary-color: #{$primary-color};
}
