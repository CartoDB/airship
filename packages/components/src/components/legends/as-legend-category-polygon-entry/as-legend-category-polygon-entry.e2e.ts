import { E2EPage, newE2EPage } from '@stencil/core/testing';

describe('as-legend-category-polygon-entry', async () => {
  describe('Rendering', async () => {
    let page: E2EPage;

    it('basic polygons', async () => {
      page = await newE2EPage({ html: `
        <as-legend-category-polygon-entry>
        </as-legend-category-polygon-entry>
      ` });

      const element = await page.find('as-legend-category-polygon-entry');
      element.setProperty('label', 'Hello there');
      element.setProperty('color', '#FABADA');
      element.setProperty('strokeColor', '#C0C1D0');
      element.setProperty('strokeStyle', 'dashed');
      await page.waitForChanges();
      expect(element.outerHTML).toMatchSnapshot();
    });
  });
});
