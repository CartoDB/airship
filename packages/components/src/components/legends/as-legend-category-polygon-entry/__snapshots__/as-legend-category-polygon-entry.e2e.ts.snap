// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as-legend-category-polygon-entry Rendering basic polygons 1`] = `"<as-legend-category-polygon-entry class=\\"hydrated\\"><div class=\\"as-legend-category-polygon-entry--wrapper\\"><div class=\\"as-legend-category--figure--wrapper\\"><div class=\\"as-legend-category-polygon-entry--box\\" style=\\"background-color: rgb(250, 186, 218); border: 1px dashed rgb(192, 193, 208);\\"></div></div><span class=\\"as-legend-category--label\\">Hello there</span></div></as-legend-category-polygon-entry>"`;
